src/OpenLoco/GameCommands/GameCommands.h

```diff
@@ -130,6 +130,13 @@ namespace OpenLoco::GameCommands
     uint32_t doCommand(GameCommand command, const registers& registers);
     bool sub_431E6A(const CompanyId_t company, Map::TileElement* const tile = nullptr);
 
+    template<typename T>
+    uint32_t doCommand(uint8_t flags, const T& args) {
```

Comment:
Think it would be better if the args parameter were in the first position. Makes it a bit more obvious what command is being run. (Sometimes the flags argument passed is quite long.)